codec-redis/src/main/java/io/netty/handler/codec/redis/RedisMessageType.java

```diff
@@ -36,9 +36,9 @@
     BLOB_ERROR((byte) '!', false),
     VERBATIM_STRING((byte) '=', false),
     BIG_NUMBER((byte) '(', true),
-    MAP((byte) '%', false),
-    SET((byte) '~', false),
-    ATTRIBUTE((byte) '|', false),
```

Comment:
isn't that a breaking change ?

Reply:
These enum is new add by me for support new types message

Reply:
If a user is relying on old enum types then he will get compile error.

Reply:
I know that, but these enum types was added in this pr, there haven't merged yet. 

To keep the naming style consistent with `ARRAY_HEADER` existed, i rename `MAP` to `MAP_HEADER`.